### PR TITLE
Production

### PR DESCRIPTION
**Pull request by thecodingpoet**
#### What does this PR do?
This PR merges the production branch to master

#### Description of Task to be completed?
To complete the inverted index application

#### How should this be manually tested?
By visiting the [website](https://indexit.herokuapp.com/)

#### Any background context you want to provide?
NA

#### Screenshots (if appropriate)

screen shot 2017-03-16 at 9 32 18 pm
screen shot 2017-03-16 at 8 00 29 pm
screen shot 2017-03-17 at 1 42 06 am
screen shot 2017-03-17 at 1 43 23 am
screen shot 2017-03-17 at 1 45 44 am
screen shot 2017-03-17 at 4 28 27 am

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+engines:
+  csslint:
+    enabled: true
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - ruby
```
Comment (ngozi-ekekwe) [inline .codeclimate.yml]: You just need to include only javascript alone, remove other languages

File: .codeclimate.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    enabled: true
+    channel: "eslint-3"
+  fixme:
+    enabled: true
+ratings:
+  paths:
+  - "**.css"
+  - "**.inc"
+  - "**.js"
+  - "**.jsx"
```
Comment (ngozi-ekekwe) [inline .codeclimate.yml]: same here, remove extensions that are not required.

File: public/js/invertedindexutil.js [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+    }
+    return uniqueWords;
+  }
+
+   /**
+   * This methods gets all the tokens in a text
+   * @param {String} text - A text of words
+   * @returns {Array} - An array consisting of all the tokens in the text
+   */
+  static Tokenize(text) {
```
Comment (ngozi-ekekwe) [inline public/js/invertedindexutil.js]: The 't' in Tokenize method should be in lowercase

**Review by ngozi-ekekwe — CHANGES_REQUESTED** [review]

File: .codeclimate.yml (hunk 1 above)
Comment (thecodingpoet) [reply .codeclimate.yml]: Done

File: .codeclimate.yml (hunk 2 above)
Comment (thecodingpoet) [reply .codeclimate.yml]: Done

**Review by ngozi-ekekwe — APPROVED** [review]
Looks Good

**Pull request closed without merging**